core/src/test/java/io/questdb/test/TelemetryTest.java

```diff
@@ -128,6 +157,34 @@ public boolean getEnabled() {
         });
     }
 
+    @Test
+    public void testTelemetryScanDirsWhenScanTimeIsHigh() throws Exception {
+        assertMemoryLeak(() -> {
+            final CairoConfiguration configuration = new DefaultTestCairoConfiguration(root) {
+                @Override
+                public @NotNull TelemetryConfiguration getTelemetryConfiguration() {
+                    return new DefaultTelemetryConfiguration() {
+                        @Override
+                        public long getMaxDbSizeEstimateTime() {
+                            return Timestamps.HOUR_MICROS;
+                        }
+                    };
+                }
+            };
+
+            try (
+                    CairoEngine engine = new CairoEngine(configuration);
+                    TelemetryJob ignored = new TelemetryJob(engine)
+            ) {
+                try (SqlCompiler compiler = engine.getSqlCompiler()) {
+                    TestUtils.printSql(compiler, new SqlExecutionContextImpl(engine, 1), TELEMETRY, sink);
+                    TestUtils.assertContains(sink, "\t1\n");
```

Comment:
Exact same assert, same test as when we do not scan.
If we had a specific value logged for timeout, we could assert for that in the output of a select like:
```
event, origin from telemetry;
```
event would be the N/A value, origin would be 1

Reply:
Alex is on hols now, it would be good for us to fix this

Reply:
i can pick it up

Reply:
redesigned the tests